woocommerce/woocommerce-android
# [Wear App] Fix Orders List cache data sorting

Summary
==========
Fix issue #12755 by updating the OrdersRepository with a sorting mechanism matching the other data sources.

How to Test
==========
1. Disable the Wear app internet connection and sever the Phone connection to force the cached data to be used.
5. Make sure that the app has already had a chance to sync data from the store at least once.
6. Open the app and navigate to the Orders List. Please verify that the list displays orders from new to old, exactly like the other data sources.


Update release notes:


## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
